auth-tokens/src/main/java/com/palantir/tokens2/auth/UnverifiedJsonWebToken.java

```diff
@@ -43,14 +41,7 @@
 public abstract class UnverifiedJsonWebToken {
```

Comment:
before we cut a release of 2.0, we may want to consider the class hierarchy around UnverifiedJsonWebToken and in particular whether we want to also support a VerifiedJsonWebToken